homeassistant/components/mqtt/__init__.py

```diff
@@ -360,10 +360,10 @@ def async_setup(hass: HomeAssistantType, config: ConfigType):
         certificate = requests.certs.where()
 
     will_message = None  # type: Optional[Message]
-    if conf[CONF_WILL_MESSAGE] is not None:
+    if conf.get(CONF_WILL_MESSAGE) is not None:
```

Comment:
Something something style changes 😝

Reply:
I don't really understand. This was a **revert** of the style change, so that it's the same as before. I.e. it was `conf.get(CONF_WILL_MESSAGE)` before...